back-end/src/common/tests/conftest.py

```diff
@@ -0,0 +1,59 @@
+# pylint: disable=unused-argument,redefined-outer-name,import-outside-toplevel,invalid-name,too-many-arguments,redefined-builtin
+import pytest
+
+from auth.models import User
+from common.models.subjects import Subject
+
+SUPERUSER_EMAIL = "superuserfortests@mail.com"
```

Comment:
You should use only one conftest.py. Now there 2: tests/conftest.py and src/conftest.py. Pls merge this 2 files